build_node/build_node_builder.py

```diff
@@ -103,24 +111,21 @@ def run(self):
                 task_log_handler = self.__init_task_logger(task_log_file)
                 self.__build_packages(task, task_dir, artifacts_dir)
                 success = True
-            except BuildError as e:
+            except BuildError:
```

Comment:
Question. Why are we excluding the error message? Maybe it's because it doesn't provide any useful info or even nothing at all? Same question applies to the same exceptions below.